### ISSUE TITLE
Data set

### ISSUE THREAD

**Issue by ghost** [zhyxun/Separating-Style-and-Content-for-Generalized-Style-Transfer#1]
Can you please provide your dataset or the txt files format?

**Comment by Johnson-yue** [comment]
@claudiuiova Did you finish train this code?

**Comment by ghost** [comment]
yes. But not on chinese characters. I tested it on english words and i had no results

**Comment by Johnson-yue** [comment]
@claudiuiova  I re-implement by using pytorch and  I also had no results.  Some different in code from paper 
johnsonyue1@gmail.com  this is my email, if you have any result , please tell me, thank you

**Comment by ecnuycxie** [comment]
@claudiuiova  can you please provide your txt files format for this project? i find it difficult to understand how to organize my dataset. my emai is "ycxie0702@126.com“. thank you very much.

**Comment by wangchen925** [comment]
can you provide your english characters set? i want to orginaize chinese characters set. my email is nian_wchen@126.com.thank you!
